tests/test_models/test_forward.py

```diff
@@ -699,3 +699,113 @@ def test_yolox_random_size():
         gt_labels=gt_labels,
         return_loss=True)
     assert detector._input_size == (64, 96)
+
+
+def test_maskformer_forward():
+    model_cfg = _get_detector_cfg(
+        'maskformer/maskformer_r50_mstrain_16x1_75e_coco.py')
```

Comment:
You can use a smaller backbone such as r18 with `_replace_r50_with_r18`
And you can also change the number of stages.